cartoframes/context.py

```diff
@@ -846,10 +846,12 @@ def safe_quotes(text, escape_single_quotes=False):
                         params=params,
                         headers={'Content-Type': 'application/json'})
 
-                # replace previous cartocss with turbo-carto processed version
-                layer.cartocss = json.loads(
-                        resp.text.split('&& cartoframes(')[1]
-                            .strip(');'))['metadata']['layers'][1]['meta']['cartocss']
+                # check if errors in cartocss (already turbo-carto processed)
+                if "errors" not in resp.text:
```

Comment:
What happens if a user is styling by a column called 'errors'?

Reply:
Using `resp.raise_for_status()` might be a more robust way to do error-checking:
http://docs.python-requests.org/en/master/api/#requests.Response.raise_for_status

Reply:
Hm, good point about the column called 'errors'. But I checked and the response status is always 200.

Reply:
When you run the map again, this is what the response.text looks like:

'/**/ typeof cartoframes === \'function\' && cartoframes({"errors":["Failed to process \\"marker-fill\\" property: unable to compute ramp, column reference \\"value\\" is ambiguous"],"errors_with_context":[{"type":"layer","message":"Failed to process \\"marker-fill\\" property: unable to compute ramp, column reference \\"value\\" is ambiguous","subtype":"turbo-carto","layer":{"id":"torque-layer0","index":1,"type":"torque","context":{"selector":"#layer","source":{"start":{"line":1,"column":321},"end":{"line":1,"column":385}}}}}]});'

Reply:
We could split up the `layer.cartocss` line into two or more steps:
1. Put `resp_dict = json.loads(resp.text.split('&& cartoframes(')[1])` into a variable
2. Since (1) is a dict, do an `if 'errors' in resp_dict: pass`, else set the processed cartocss to `layer.cartocss`

Reply:
I guess `if 'errors' not in resp_dict: layer.cartocss = ...` is more concise